sites/website/sidebars.js

```diff
@@ -83,6 +83,8 @@ module.exports = {
             type: "category",
             label: "API Reference",
             items: [
+                "api/fast-animation",
+                "api/fast-colors",
                 "api/fast-element",
                 "api/fast-foundation",
                 "api/fast-components",
```

Comment:
Since we have more APIs, maybe we should move fast-components up to that it's fully alphabetized?